parlai/tasks/convai2/agents.py

```diff
@@ -69,6 +69,18 @@ def __init__(self, opt, shared=None):
         super().__init__(opt, shared)
 
 
+class LimitedSelfOriginalTeacher(SelfOriginalTeacher):
```

Comment:
can we just do --num-epochs .001 instead of making a new teacher?

Reply:
This does not seem to apply for the final eval of the train loop (on valid and test set) does it?

Reply:
Oh jeez the damn validation; you're right.

Reply:
I’m going to make a flag to help us with that. I ran into this exact issue two weeks ago

Reply:
cf #1606